synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp

```diff
@@ -164,6 +175,26 @@ ffmpeg_trgt::set_rend_desc(RendDesc *given_desc)
 bool
 ffmpeg_trgt::init(ProgressCallback *cb=NULL)
 {
+
+	if(!SoundProcessor::subsys_init())
+		throw std::runtime_error(_("Unable to initialize subsystem \"Sound\""));
```

Comment:
Wouldn't be better to report error in `cb` and return false, instead of throwing an exception?

Reply:
Absolutely! Did the changes now. ^__^